app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,24 @@
+class InvitationsController < ApplicationController
+  before_filter :require_user
+  def new
+  	@invitation = Invitation.new
+  end
+
+  def create
+  	@invitation = Invitation.new(invitation_params.merge!(inviter_id: current_user.id))
+    
+    if @invitation.save
+      MyflixMailer.send_invitation_email(@invitation).deliver
+      flash[:success] = "You have successfully invited #{@invitation.recipient_name}"
+    	redirect_to new_invitation_path
+    else
+      flash[:error] = "Please check your inputs"
```

Comment:
When you render template, you should use `flash.now`.

HOMEWORK: Figure out why you should use `.now` method and explain.